ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2362,6 +2362,39 @@ private enum UnityLayers {
     reset_rsas_target_ = true;
   }
 
+  private static void SetIntegrators(IntPtr plugin,
+                                     ConfigNode gravity_model_config) {
+   if (gravity_model_config == null) {
+     return;
+   }
+   var ephemeris_parameters =
```

Comment:
I would actually like it if `sol_gravity_model.cfg` had explicit configurations for the integrators.  This would not be journal-incompatible as we use KSP for the journal, it would serve as an example for users, and it would make things more explicit.  Doesn't need to be in this PR, but maybe a TODO?

Reply:
As discussed, this should actually be in a separate config node (and correspondingly in a separate proto in astronomy).

Changed the C#, opened #1669 to track this.